From de4704d6056558068f9eb2b85ccf6138acb4395a Mon Sep 17 00:00:00 2001
From: Michael Cook <code@mdcook.net>
Date: Fri, 29 Dec 2017 22:56:19 -0800
Subject: [PATCH 049/249] fix syntax for new liquid version

- Liquid Warning: Liquid syntax error (line 1): Expected dotdot
  but found pipe in "{{(site.pages | sort: 'title') }}" in <snip>

Change-Id: I000f455122b2f89f261c65e21f9b42eb50042ae6
---
 pages.md                            | 2 +-
 pages/build_guides.md               | 2 +-
 pages/how-tos.md                    | 2 +-
 pages/install_guides.md             | 2 +-
 pages/meta/developer_information.md | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/pages.md b/pages.md
index 782d476..25667ea 100644
--- a/pages.md
+++ b/pages.md
@@ -6,7 +6,7 @@ tags:
  - noindex
 ---
 
-{% assign sorted_pages = (site.pages | sort: 'title') %}
+{% assign sorted_pages = site.pages | sort: 'title' %}
 {% assign ignore_tags = "noindex|how-to" | split: "|" %}
 
 {% for page in sorted_pages %}
diff --git a/pages/build_guides.md b/pages/build_guides.md
index 42f7c04..ea8e6ca 100644
--- a/pages/build_guides.md
+++ b/pages/build_guides.md
@@ -4,7 +4,7 @@ sidebar: home_sidebar
 permalink: build_guides.html
 ---
 
-{% assign sorted_pages = (site.pages | sort: 'title') %}
+{% assign sorted_pages = site.pages | sort: 'title' %}
 
 {% for page in sorted_pages %}
 {% if page.title and page.url and page.url contains "/devices/" and page.url contains "/build" %}
diff --git a/pages/how-tos.md b/pages/how-tos.md
index 9449b65..2eb3225 100644
--- a/pages/how-tos.md
+++ b/pages/how-tos.md
@@ -5,7 +5,7 @@ permalink: how-tos.html
 tags:
 ---
 
-{% assign sorted_pages = (site.pages | sort: 'title') %}
+{% assign sorted_pages = site.pages | sort: 'title' %}
 
 {% for page in sorted_pages %}
 {% if page.title and page.url and page.tags contains "how-to" %}
diff --git a/pages/install_guides.md b/pages/install_guides.md
index 05918ac..b650d2d 100644
--- a/pages/install_guides.md
+++ b/pages/install_guides.md
@@ -4,7 +4,7 @@ sidebar: home_sidebar
 permalink: install_guides.html
 ---
 
-{% assign sorted_pages = (site.pages | sort: 'title') %}
+{% assign sorted_pages = site.pages | sort: 'title' %}
 
 {% for page in sorted_pages %}
 {% if page.title and page.url and page.url contains "/devices/" and page.url contains "/install" %}
diff --git a/pages/meta/developer_information.md b/pages/meta/developer_information.md
index 4a137b8..397a5f6 100644
--- a/pages/meta/developer_information.md
+++ b/pages/meta/developer_information.md
@@ -10,7 +10,7 @@ So, you've decided you want to contribute to the Lineage software. Awesome! This
 
 These are pages you might want to check out for the information you are searching
 
-{% assign sorted_pages = (site.pages | sort: 'title') %}
+{% assign sorted_pages = site.pages | sort: 'title' %}
 
 {% for page in sorted_pages %}
 {% if page.tags contains "internal" %}
-- 
2.17.1

